Frontend/ui-common/src/DecodeUtil/DecodeUtil.js

```
@@ -69,3 +71,22 @@ export const unsafeGet = function (key,obj) {
+  }
+};
+
+export const getFibers = () => {
+  return () => {
+    return fibers;
+  }
+};
+
+export const resetFibers = () => {
+  return () => {
+    fibers = [];
```

Biome (1.9.4)

[error] Can't assign fibers because it's a constant

This is where the variable is defined as constant

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)